### ISSUE TITLE
breaking change

### ISSUE THREAD

**Issue by akosenkov** [huggingface/transformers#1246]
Great job, just in case it went unnoticed:
from revision 995e38b7af1aa325b994246e1bfcc7bf7c9b6b4f
to revision 2c177a87eb5faab8a0abee907ff75898b4886689

examples are broken due to changed orders of parameters in 
pytorch_transformers/modeling_bert.py

```
<     def forward(self, input_ids, token_type_ids=None, attention_mask=None, labels=None,
<                 position_ids=None, head_mask=None):
<         outputs = self.bert(input_ids, position_ids=position_ids, token_type_ids=token_type_ids,
<                             attention_mask=attention_mask, head_mask=head_mask)
---
>     def forward(self, input_ids, attention_mask=None, token_type_ids=None,
>                 position_ids=None, head_mask=None, labels=None):
>
```

**Comment by thomwolf** [comment]
Indeed. What do you mean by "examples"? The docstrings examples?

**Comment by akosenkov** [comment]
Most of examples folder.
In particular run_swag.py and lm finetuning scripts.

**Comment by thomwolf** [comment]
Indeed, I've fixed and cleaned up the examples in 8334993 (the lm finetuning examples are now replaced by `run_lm_finetuning`). Also indicated more clearly which examples are not actively maintained and tested.
